gen/dcos-config.yaml

```diff
@@ -132,13 +132,13 @@ package:
         "replicated_log_dir":"/var/lib/dcos/mesos/master/",
         "network": {{ dcos_overlay_network }}
        }
-  - path: /etc/dcos/network/cni/dummy.cni
+  - path: /etc/dcos/network/cni/dummy.conf
```

Comment:
We actually don't need `dummy.conf` any more. We can remove it. It was introduced in the initial days of the `network/cni` isolator since the isolator would read configuration only at agent restart. This has however been fixed and the isolator picks up config files as and when they are added during container launch.